pymc_marketing/clv/models/basic.py

```diff
@@ -32,6 +33,7 @@
 class CLVModel(ModelBuilder):
     _model_type = "CLVModel"
 
+    @validate_call(config=ConfigDict(arbitrary_types_allowed=True))
```

Comment:
What requires this?

Reply:
Because by default it does not support `pd.DataFrame` types. We could also use `IsInstanceOf` but I was not sure if this makes the function signature less clear as pandas DataFrames is something we use very often

Reply:
Do you have an example of how this looks in the docs then?